common/crypto/dkg/src/bte/mod.rs

```diff
@@ -305,7 +234,7 @@ pub struct Params {
 pub fn setup() -> Params {
     let f0 = hash_g2(b"f0", SETUP_DOMAIN);
 
-    let fs = (1..=MAX_EPOCHS_EXP)
+    let fs = (1..=1)
```

Comment:
I'm surprised clippy is not complaining here that you're iterating over a single element : )